mrmustard/lab_dev/states/gaussian_state.py

```diff
@@ -76,22 +74,22 @@ def __init__(
                 update_symplectic,
             )
         )
-
         self._representation = self.from_ansatz(
             modes=modes,
             ansatz=PolyExpAnsatz.from_function(
                 fn=triples.gket_state_Abc, symplectic=self.parameters.symplectic
             ),
         ).representation
 
-    def _getitem_builtin(self, modes: set[int] | Sequence[int]):
+    def __getitem__(self, idx: int | Collection[int]):
```

Comment:
I think we should use `int | Sequence` because `Collection` is not ordered. I think it was wrong before your change to type hint it with `set`.